public/style.css

```
@@ -15,3 +15,7 @@
 ul {
   list-style: none;
 }
+
+.hide{
+  display:none;
```

Add a space between the property and the value, for code style consistency. 

```suggestion
  display: none;
```